flow/src/test/java/flow/FlowTest.java

```diff
@@ -90,7 +94,8 @@
         flow.setDispatcher(this);
       }
 
-      @Override public void dispatch(@NonNull Traversal traversal, @NonNull TraversalCallback onComplete) {
+      @Override
+      public void dispatch(@NonNull Traversal traversal, @NonNull TraversalCallback onComplete) {
```

Comment:
If leaving on the same line, the line becomes more than 100 characters.
I'm actually using the [square code style](https://github.com/square/java-code-styles), and formatting is done automatically